src/include/ipc4/module.h

```diff
@@ -64,7 +64,7 @@ union ipc4_extended_param_id {
 		uint32_t parameter_type     : 8;
 		uint32_t parameter_instance : 24;
 	} part;
-} __packed __aligned(4);
+} __attribute__((packed, aligned(4)));
```

Comment:
I noticed kernel use left expression for alignment, and sof fw use right ones, any history on this?

Reply:
"__packed" is a linux kernel shortcut, not necessarily available in generic toolchains (like for SOF plugin).